_rules/meta-viewport-b4f0c3.md

```diff
@@ -0,0 +1,256 @@
+---
+id: b4f0c3
+name: meta viewport does not prevent zoom
+rule_type: atomic
+description: |
+  This rule checks that the `meta` element is not used to block the user agent ability to zoom.
+accessibility_requirements:
+  wcag21:1.4.4: # Resize text (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+authors:
+  - Jey Nandakumar
+  - Audrey Maniez
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint
+  # (used with `npm test` to ensure validity of code snippets)
+  - 'title-require'
+---
+
+## Applicability
+
+The rule applies to each [`<meta>`](https://www.w3.org/TR/html52/document-metadata.html#the-meta-element) element with a [`name`](https://www.w3.org/TR/html52/document-metadata.html#element-attrdef-meta-name) attribute whose value is set to [`viewport`](https://www.w3.org/TR/css-device-adapt-1/#viewport-meta) and has a [`content`](https://www.w3.org/TR/html52/document-metadata.html#dom-htmlmetaelement-content) attribute.
+
+## Expectation
+
+The [`content`](https://www.w3.org/TR/html52/document-metadata.html#dom-htmlmetaelement-content) attribute, whose value is mapped to a list of property/value pairs in a user-agent specific manner, does not:
+
+- specify the property `user-scalable` with a value of `no`; nor
+- specify the property `maximum-scale` with a value of less than 2
+
+**Note:** For a known popular browser, the content attribute is mapped to property/value pairs according to the following algorithm: https://www.w3.org/TR/css-device-adapt-1/#parsing-algorithm
+
+## Assumptions
+
+- The [page](https://www.w3.org/TR/WCAG21/#dfn-web-page-s) has [visible](#visible) [content](https://www.w3.org/TR/WCAG21/#dfn-content).
+
+## Accessibility Support
+
+- Interpretation of `<meta name="viewport">` may vary with browsers, meaning browsers may override `user-scalable=no` and `maximum-scale=1` values to support gestures like pinch and zoom based interaction on permissible devices, thereby causing inconsistent behavior.
```

Comment:
Can you add an explanation of what this statement means for this rule?

Reply:
I agree with you. This sentence is too generic & does not cover the core of what this rule tries to establish. I was having it as a caveat to cater for cases where these values are set to achieve other behaviour. I will remove this.